src/components/AddressOrderCard/index.js

```diff
@@ -0,0 +1,272 @@
+import React, {useState, useEffect} from 'react'
+import Form from "react-bootstrap/Form";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import { signUp } from "../../store/clients/actions";
+import { selectToken } from "../../store/clients/selectors";
+import { useDispatch, useSelector } from "react-redux";
+import { useHistory, Link } from "react-router-dom";
+import { Col } from "react-bootstrap";
+import './index.css'
+
+export default function AddressOrderCard() {
+
+    const [typeOrder, setTypeOrder] = useState("")
+    const [name, setName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [middleName, setMiddleName] = useState("")
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
Already commented a lot on the backend on this but you should take care of keeping your code clean and properly indented